app/components/Account/AccountOverview.jsx

```diff
@@ -3,7 +3,7 @@ import Immutable from "immutable";
 import Translate from "react-translate-component";
 import TotalBalanceValue from "../Utility/TotalBalanceValue";
 import AssetName from "../Utility/AssetName";
-import MarginPositions from "./MarginPositions";
+import CollateralTable from "./CollateralTable";
```

Comment:
I'd suggest to be consistent in the naming. Can this be MarginPositionTable?

Reply:
sure